source/api_documentation/call_ingestion_api/index.rst

```diff
@@ -0,0 +1,321 @@
+Call Ingestion API
+=================
+
+The Call Ingestion API is used to submit call details along with a recording URL for calls that were not live-connected and routed through the Invoca Telephony platform.
+
+Calls submitted through this API are available to all post-call processing systems. This includes the Invoca transcription engine, Signal AI, as well as the Dashboard and Reporting system. 
+
+Invoca post-call processing services are applied to calls based on network and campaign settings.  The campaign for the call is determined by the *advertiser_campaign_id_from_network* parameter in the API.
+
+Additionally, Signals and Custom Data related to the call can be sumbitted during the initial call ingestion step.  Please note that Signals and Custom Data can also be applied in the future using the :doc:`../signal_api/index`.
+
+Signals can be any boolean value (e.g. sale, quote, etc), and Custom Data can be any alpha-numeric value (e.g. account type, customer quality score, etc).
+
+Response Codes
+--------------
+
+Remember to check the HTTP status code returned. This helps greatly when debugging.
+
+.. list-table::
+  :widths: 8 40
+  :header-rows: 1
+  :class: parameters
+
+
+  * - Status Code
+    - Meaning
+
+  * - 201 Created
+    - A new call creation request was successfully created.
+
+  * - 202 Accepted
+    - The call creation request has already been received and is currently being processed.
```

Comment:
Should we append "or has already been processed", since we will return a 202 if the import has succeeded (since the ingestion request table isn't concerned with whether it finished or not)

Reply:
good catch @kamerynn fixed...